client/src/App.js

```diff
@@ -809,6 +865,86 @@ class App extends Component {
           />
         </>
         )
+      case COUNTRY_AMBASSADOR:
+        return (
```

Comment:
Personally, I find this whole chunk a bit difficult to read.. Wondering if there's a way to generalize these cases to simplify this code?

Reply:
Yeah similar to the comment I made further down, I kinda wonder if you could pass a small set of props to a subcomponent and generalize this code in a really reusable way

Reply:
Hmm, definitely agree. The App.js component is unwieldily clunky, I'll take a bit to extract this piece out